src/App.tsx

```diff
@@ -15,47 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  let copyGoods = [...goodsFromServer];
+  const [sortType, setsortType] = useState(SortType.NONE);
+  const [isReversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [sortType, setSortType] = useState(SortType.NONE);
  const [isReversed, setIsReversed] = useState(false);
```
SetState function should be named according to the noun name. 
Also. don't forget about camelCase